src/settings/NoticePolicy/components/DetailSections/GeneralSection/GeneralSection.js

```diff
@@ -27,41 +27,43 @@ const GeneralSection = (props) => {
     : <FormattedMessage id="ui-circulation.settings.loanPolicy.no" />;
 
   return (
-    <Accordion
-      open={isOpen}
-      id="generalInformation"
-      onToggle={onToggle}
-      label={<FormattedMessage id="ui-circulation.settings.loanPolicy.generalInformation" />}
-    >
-      <Metadata
-        connect={connect}
-        metadata={metadata}
-      />
-      <Row>
-        <Col xs={12}>
-          <KeyValue
-            label={<FormattedMessage id="ui-circulation.settings.loanPolicy.policyName" />}
-            value={policyName}
-          />
-        </Col>
-      </Row>
-      <Row>
-        <Col xs={12}>
-          <KeyValue
-            label={<FormattedMessage id="ui-circulation.settings.noticePolicy.active" />}
-            value={active}
-          />
-        </Col>
-      </Row>
-      <Row>
-        <Col xs={12}>
-          <KeyValue
-            label={<FormattedMessage id="ui-circulation.settings.loanPolicy.policyDescription" />}
-            value={policyDescription}
-          />
-        </Col>
-      </Row>
-    </Accordion>
+    <div data-test-notice-policy-detail-general-section>
+      <Accordion
+        open={isOpen}
+        id="generalInformation"
+        onToggle={onToggle}
+        label={<FormattedMessage id="ui-circulation.settings.loanPolicy.generalInformation" />}
+      >
+        <Metadata
+          connect={connect}
+          metadata={metadata}
+        />
+        <Row>
+          <Col xs={12} data-notice-policy-name>
```

Comment:
Since you have snippets where you place attributes on its own lines, please follow it here as well.